packages/cactus-plugin-ledger-connector-corda/src/main-server/kotlin/gen/kotlin-spring/src/main/kotlin/org/hyperledger/cactus/plugin/ledger/connector/corda/server/model/NetworkMapV1Request.kt

```diff
@@ -1,39 +0,0 @@
-package org.hyperledger.cactus.plugin.ledger.connector.corda.server.model
```

Comment:
@elenaizaguirre Question: Why were the contents of this file removed? I thought the code generator would fill this based on the openapi.json.

Reply:
@petermetz The object definition in `openapi.json` is empty, it simply defines it as object type. I don't know when this file was generated but if you delete the `model` folder and generate it with the `generate-server` script in `package.json` (it should probably also take care of deleting that folder) you will see that `NetworkMapV1Request` and `NetworkMapV1Response` are not generated, but the new connector is working in all tests.

Reply:
@elenaizaguirre My bad, this is an issue with the build itself then, sorry. I created this for us to follow up on the issue later on more: https://github.com/hyperledger/cactus/issues/1559